### PR TITLE
CI 781xx relnotes per DOC-2423

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/continuous-integration.md [hunk 1]
```diff
@@ -11,13 +11,40 @@ Harness Continuous Integration is updated regularly. Review the notes below for
 Harness deploys updates progressively to different Harness SaaS clusters. You can identify the cluster hosting your account in your Account Overview page. The features and fixes in the release notes may not be available in your cluster immediately.
 :::
 
+## January 5, 2023, version 78100
```
Comment (KimberlyFields) [inline release-notes/continuous-integration.md:14]: 78100 is not the final version number. We will get the final version number from QA after the release sign-off on January 10th or 11th.

File: release-notes/continuous-integration.md [hunk 2]
```diff
@@ -11,13 +11,40 @@ Harness Continuous Integration is updated regularly. Review the notes below for
...
+
+### Fixed issues
+
+* Fixed an issue in the onboarding UI. In some cases, the web UI did not connect with the specified git account immediately after OAuth setup. (CI-6518)
+
+* You can now use codebase fields as expressions. Previously, when the codebase repository name was defined by `<+pipeline.name>`, the built-in variables `<+codebase.branch>` and `<+codebase.commitSha>` returned null. (CI-6478, ZD-38122, ZD-38241)
+
+* Fixed an issue that prevented Azure connectors from connecting to repos with URLs that contained spaces or `%20`. (CI-6465)
+
+* Fixed an issue where CI reports an upload to an S3 as successful even if the upload failed. (CI-6420, ZD-37931)
```
Comment (KimberlyFields) [inline release-notes/continuous-integration.md:32]:
```suggestion
* Fixed an issue where CI reported an upload to an S3 as successful even if the upload failed. (CI-6420, ZD-37931)
```

File: release-notes/continuous-integration.md [hunk 3]
```diff
@@ -11,13 +11,40 @@ Harness Continuous Integration is updated regularly. Review the notes below for
...
+
+* Fixed an issue in the onboarding UI where the Select Your Repository progress bar was incorrect. (CI-6335)
```
Comment (KimberlyFields) [inline release-notes/continuous-integration.md:34]:
```suggestion
* Fixed an issue in the onboarding UI where the **Select Your Repository** progress bar was incorrect. (CI-6335)
```

File: release-notes/continuous-integration.md [hunk 4]
```diff
@@ -11,13 +11,40 @@ Harness Continuous Integration is updated regularly. Review the notes below for
...
+
+* Fixed an issue where a build does not run when a user clicks Run in the Pipeline Editor UI. To run the build, the user needed to go to the YAML editor, save, and then run. (CI-6239)
```
Comment (KimberlyFields) [inline release-notes/continuous-integration.md:36]:
```suggestion
* Fixed an issue where a build did not run when a user selected **Run** in the **Pipeline Editor** UI. To run the build, the user needed to go to the YAML editor, save, and then run. (CI-6239)
```

File: release-notes/continuous-integration.md (hunk 2 above)
Comment (rajatharanganath) [reply release-notes/continuous-integration.md:32]: * Fixed an issue where CI reported an upload to a S3 bucket as successful even if the upload failed. (CI-6420, ZD-37931)

**Review by douglas-j-bothwell — COMMENTED** [review]
Implemented edits from Kim and Rajatha, will now merge PR.

**Pull request merged**